### ISSUE TITLE
[BUG] - Can't find export (react-aria)

### ISSUE THREAD

**Issue by novacoole** [nextui-org/nextui#3535]
### NextUI Version

2.4.6

### Describe the bug

Followed automatic installation steps [here](https://nextui.org/docs/guide/installation), selected 'Vite', and added all components.

Then i imported these components:
```import { Card, Button, Spacer, CardFooter, CardHeader, CardBody } from '@nextui-org/react';```

and received following compilation error:
```
 [ERROR] No matching export in "node_modules/@react-aria/table/node_modules/@react-aria/utils/dist/import.mjs" for import "getSyntheticLinkProps"

    node_modules/@react-aria/table/dist/useTableRow.mjs:3:8:
      3 │ import {getSyntheticLinkProps as $2K6sg$getSyntheticLinkProps, mergeProps as $2K6sg$mergeProps} from "@react-ar...
        ╵         ~~~~~~~~~~~~~~~~~~~~~
```

The issue has been raised with react-aria [here](https://github.com/adobe/react-spectrum/issues/6750) but as i'm also using NextUI (same as creator of that issue), I have a suspicion it is a packaging issue from NextUI.

### Steps to Reproduce the Bug or Issue

1. `nextui init my-nextui-app`
2. select vite
3. `nextui add --all`
4. import `Card, Button, Spacer, CardFooter, CardHeader, CardBody` from next-ui-org/react and render them
5. run dev server

### Expected behavior

No compilation issues.

### Operating System Version

Windows 11 (Ubuntu 24.04 WSL)

### Browser

Chrome

**Comment by theonlyway** [comment]
NextJS trace seems a bit more helpful

```
./node_modules/@react-aria/table/dist/useTableRow.mjs
Attempted import error: 'getSyntheticLinkProps' is not exported from '@react-aria/utils' (imported as '$2K6sg$getSyntheticLinkProps').

Import trace for requested module:
./node_modules/@react-aria/table/dist/useTableRow.mjs
./node_modules/@react-aria/table/dist/import.mjs
./node_modules/@nextui-org/table/dist/chunk-IZM76DYO.mjs
./node_modules/@nextui-org/table/dist/index.mjs
./components/admin/rootLayout.tsx
```

**Comment by theonlyway** [comment]
Also getting this as well

```
./node_modules/@react-aria/table/dist/useTableRow.mjs
Attempted import error: 'getSyntheticLinkProps' is not exported from '@react-aria/utils' (imported as '$2K6sg$getSyntheticLinkProps').

Import trace for requested module:
./node_modules/@react-aria/table/dist/useTableRow.mjs
./node_modules/@react-aria/table/dist/import.mjs
./node_modules/@nextui-org/table/dist/chunk-IZM76DYO.mjs
./node_modules/@nextui-org/table/dist/index.mjs
./components/admin/rootLayout.tsx
```

**Comment by davidbonan** [comment]
Same error

**Comment by nmthangdn2000** [comment]
Same error
![CleanShot 2024-07-23 at 15 00 46@2x](https://github.com/user-attachments/assets/6ef83c76-6a5b-437b-a421-2e4baa22dfe5)

**Comment by abhinandan-verma** [comment]
Actually the  problem is that `getSyntheticLinkProps` has been either changed to `useSyntheticLinkProps`  in `@react-aria/utils`, so that 

<img width="1272" alt="Screenshot 2024-07-23 at 1 25 45 PM" src="https://github.com/user-attachments/assets/ba067045-478d-4996-894a-e253aa59234e">


<img width="1012" alt="Screenshot 2024-07-23 at 1 32 18 PM" src="https://github.com/user-attachments/assets/75164fe0-0346-4db3-91fc-e177b6a36dbc">



I will solve the bug, can anyone help me where can I find this file [react-aria](https://github.com/adobe/react-spectrum)

**Comment by davidbonan** [comment]
@abhinandan-verma Here : 

useSyntheticLinkProps : 
https://github.com/adobe/react-spectrum/blob/139374acb38dd7fb93bc62b346db5a494feecca3/packages/%40react-aria/utils/src/openLink.tsx#L149


https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/table/src/useTableRow.ts

**Comment by LeNgocMinh2704** [comment]
![Screenshot 2024-07-23 153135](https://github.com/user-attachments/assets/de640f37-041b-49e2-97f3-5bdff57e7c9d)
Getting the same bug with latest Nextjs

**Comment by abhinandan-verma** [comment]
@LeNgocMinh2704  can you use ` Next.js 14.2.4 (turbo)` version for now,  as with this version I am not getting any  bug

**Comment by LeNgocMinh2704** [comment]
> @LeNgocMinh2704 can you use ` Next.js 14.2.4 (turbo)` version for now, as with this version I am not getting any bug

Running with dev -- turbo works just fine, but when building for production using yarn build, I get this error.

**Comment by abhinandan-verma** [comment]
@LeNgocMinh2704  and all can you all retry the set up from start (if possible), as I am not getting any error now both using `nextjs` and `vite`

<img width="407" alt="Screenshot 2024-07-23 at 2 31 07 PM" src="https://github.com/user-attachments/assets/d56134ae-f07a-4315-b547-3bf5c57b69ca">

**Comment by Mohammed-alhasan** [comment]
Same error

**Comment by ArielBenichou** [comment]
+1

**Comment by wingkwong** [comment]
UPD: That above PR was experimental and it got merged by mistake. We've rollbacked it. We'll wait for RA team's updates and at the same time we're trying to upgrade to RA latest and resolve the build issues.

**Comment by PONZACENIOW** [comment]
⨯ ./node_modules/@react-aria/table/dist/import.mjs
Attempted import error: 'getSyntheticLinkProps' is not exported from '@react-aria/utils' (imported as '$lJcFS$getSyntheticLinkProps').

**Comment by SebasAas** [comment]
+1

**Comment by devongovett** [comment]
This should be fixed in `@react-aria/utils` v3.25.1. Sorry for the bug and thanks for your patience.

**Comment by wingkwong** [comment]
Closing - fixed in https://github.com/adobe/react-spectrum/pull/6755 by RA team.
